docs/source/user_guide/instructions.rst

```diff
@@ -0,0 +1,161 @@
+***************************************
+Instructions
+***************************************
+
+Home View
+==============================
+
+This is the home page of the Birdspotter app and is where you can navigate the datasets that you can access.
+
+.. _home_view:
+.. figure:: static/home_view.png
+   :alt: "Example of the home view"
+   :align: center
+
+   : Example of the home view
+   
+Dataset Selection and Navigating to a View
+##########################################
+
+1a. To navigate to the map view, click on the left icon under the "View" column of the chosen dataset row.
+
+.. _map_view_highlight:
+.. figure:: static/map_view_highlight.png
+   :alt: "Selection for map view"
+   :align: center
+   
+   : Selection for map view
+   
+1b. To navigate to the data view, click on the right icon under the "View" column of the chosen dataset row.
+
+.. _data_view_highlight:
+.. figure:: static/data_view_highlight.png
+   :alt: "Selection for data view"
+   :align: center
+   
+   : Selection for data view
+   
+2. To navigate back, click on the back arrow or Birdspotter logo.
+
+
+Dataset Actions
+###############
+
+1. Select the "Actions" dropdown in the Actions column of the chosen dataset row.
+
+Modify Dataset Metadata:
+*************************
+
+.. note:: Modifying Metadata is only available for datasets that you own.
+
+1. Select "Edit Dataset".
+
+2. Change the metadata to match what you would like to be displayed.
+
+
+
+Queue Analysis of a Dataset
+*****************************
+
+.. warning::  Queueing is not yet implemented
+
+1. Select “Queue Analysis”
+2. Select the algorithm to queue.
+
+
+Export Dataset
+********************
+
+.. warning::  Exporting is not yet implemented
+   
+1. Select "Export".
+
+Share Dataset
+********************
+
+.. warning::  Exporting is not yet implemented
+   
+1. Select "Share".
+
+Data View
+==============================
+
+This view allows registered users to view the data of their own and publically available datasets.
+
+.. _data_view:
+.. figure:: static/data_view.png
+   :alt: "Example Data view for Registered User"
+   :align: center
+
+   : The data view as it may appear to a Registered User
+
+Map View
+==============================
+
+
+This view allows registered users to view the contents of the datasets that they have access to overlaid on a world map.
+
+.. _map_view:
+.. figure:: static/map_view.png
+   :alt: "Example Map view for Registered User"
+   :align: center
+
+   : The map view as it may appear to a Registered User
+
+Importing Files
+==============================
+
+This function allows you to import .tif and .zip files.
```

Comment:
(nitpick) In the overview we say `.tiff`, They are the same thing so I am fine leaving it as.

Reply:
Turns out, .tif is correct, so I've updated the overview in an upcoming commit.